src/landOnFunctions.js

```diff
@@ -1,57 +1,67 @@
-export {standardProperty, railroad, utility, eventCard, noEvent, go, goToJail, incomeTax, luxuryTax};
+export {
+    standardProperty,
+    railroad,
+    utility,
+    eventCard,
+    noEvent,
+    go,
+    goToJail,
+    incomeTax,
+    luxuryTax,
+};
 
 function standardProperty(activePlayer, emitter){
-  if(this.ownerID == null){
-    emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
-  }
-  else{
-    emitter.emit(`promptPayment${activePlayer.socketID}`, this.ownerID, activePlayer, this.rent[houses]);
-  }
+    if(this.ownerID == null){
```

Comment:
Make sure you use `===` instead of `==`. `==` will choose the least common type of the two and compare that.  Basically, never use `==`.  `===` is more safe.